setup.py

```diff
@@ -294,6 +294,11 @@ def get_long_description():
             ]
         },
         install_requires=["apptools", "setuptools", "traits"],
+        extras_require={
```

Comment:
I maybe should add `"docs": ["enthought-sphinx-theme", "Sphinx!=3.2.0"]` here as well.  Additionally the keys here could be more specific.

Reply:
Tthat sounds like a good idea to me. The keys don't need to be anymore specific that what they are in traits.